### PR TITLE
Feature/redesign

### PR DESCRIPTION
**Pull request by jmhrpr**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Site settings
-title: Cr0wn
-description: "The UK's Top CTF Team"
+title: the cr0wn
+description: "the UK’s top competitive cyber security team"
```
Comment (Josh-00FF00) [inline _config.yml]: I think the current ones are clearer, I like the old tag line since I changed from this already

File: _includes/header.html [hunk 2]
```diff
@@ -77,14 +75,6 @@ <h1 class="panel-cover__title panel-title">{{ site.title }}</h1>
               </li>
             {% endif %}
 
-            <!-- RSS -->
```
Comment (Josh-00FF00) [inline _includes/header.html:80]: I don't see a reason to remove RSS?

File: images/favicons/browserconfig.xml [hunk 3]
```diff
@@ -1,12 +1,2 @@
...
-      <square70x70logo src="/images/favicons/mstile-70x70.png"/>
-      <square150x150logo src="/images/favicons/mstile-150x150.png"/>
-      <square310x310logo src="/images/favicons/mstile-310x310.png"/>
-      <wide310x150logo src="/images/favicons/mstile-310x150.png"/>
-      <TileColor>#00aba9</TileColor>
-    </tile>
-  </msapplication>
-</browserconfig>
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig><msapplication><tile><square70x70logo src="/ms-icon-70x70.png"/><square150x150logo src="/ms-icon-150x150.png"/><square310x310logo src="/ms-icon-310x310.png"/><TileColor>#ffffff</TileColor></tile></msapplication></browserconfig>
```
Comment (Josh-00FF00) [inline images/favicons/browserconfig.xml]: Could you format this file?

File: images/favicons/manifest.json [hunk 4]
```diff
@@ -1,41 +1,41 @@
...
-		},
-		{
-			"src": "\/images\/favicons\/android-chrome-192x192.png",
-			"sizes": "192x192",
-			"type": "image\/png",
-			"density": 4
-		}
-	]
-}
+ "name": "App",
```
Comment (Josh-00FF00) [inline images/favicons/manifest.json]: Better name? Something like Cr0wn Theme?

File: _includes/header.html [hunk 5]
```diff
@@ -3,24 +3,22 @@
...
 
-<header class="panel-cover" style="background-image: url({{ site.baseurl }}/images/cover.jpg)">
+<header class="panel-cover">
   <div class="panel-main">
 
     <div class="panel-main__inner panel-inverted">
     <div class="panel-main__content">
         <a href="{{ site.baseurl }}/" title="link to home of {{ site.title }}">
-          <img src="{{ site.baseurl }}/images/profile.png" class="user-image" alt="My Profile Photo">
+          <img src="{{ site.baseurl }}/images/logo.png" class="crown-logo" alt="My Profile Photo">
```
Comment (Josh-00FF00) [inline _includes/header.html:12]: This doesn't exist so the image doesn't load

File: _includes/header.html [hunk 6]
```diff
@@ -3,24 +3,22 @@
...
-        <hr class="panel-cover__divider">
         <p class="panel-cover__description">{{ site.description }}</p>
-        <hr class="panel-cover__divider panel-cover__divider--secondary">
 
         <div class="navigation-wrapper">
 
           <nav class="cover-navigation cover-navigation--primary">
             <ul class="navigation">
-              <li class="navigation__item"><a href="{{ site.baseurl }}/#blog" title="link to {{ site.title }} blog" class="blog-button">Blog</a></li>
+              <li class="navigation__item"><a href="{{ site.baseurl }}/#blog" title="link to {{ site.title }} blog" class="blog-button">writeups</a></li>
```
Comment (Josh-00FF00) [inline _includes/header.html]: Good shout on this

File: _layouts/default.html [hunk 7]
```diff
@@ -10,7 +10,7 @@
         {{ content }}
       </div>
 
-      {% include footer.html %}
     </div>
+  {% include footer.html %}
```
Comment (Josh-00FF00) [inline _layouts/default.html]: image

Your footer changes leave a gap here, can we fix that?

File: _includes/header.html (hunk 6 above)
Comment (Josh-00FF00) [reply _includes/header.html]: Could you change the tag from `#blog` to `#writeups`?

File: _sass/uno.scss [hunk 8]
```diff
@@ -1148,9 +1149,11 @@ hr {
 	color: #c7c7c7; }
 
 .post-meta__date {
-	margin-right: .5em; }
+	margin-right: 0;
+	text-transform: lowercase; }
```
Comment (Josh-00FF00) [inline _sass/uno.scss]: I don't think we should lowercase dates `Jun` vs `jun`

File: _sass/uno.scss [hunk 9]
```diff
@@ -1148,9 +1149,11 @@ hr {
...
 
 .post-meta__tags {
+	padding: 0 !important;
```
Comment (Josh-00FF00) [inline _sass/uno.scss]: Can you make this change in a different place without `!important`?

File: _sass/uno.scss [hunk 10]
```diff
@@ -1190,27 +1193,33 @@ hr {
 	vertical-align: middle; }
 
 .panel-title {
-	margin: 0 0 5px 0;
```
Comment (Josh-00FF00) [inline _sass/uno.scss:1193]: Kinda liked a bit of spacing between these

File: _sass/uno.scss [hunk 11]
```diff
@@ -1190,27 +1193,33 @@ hr {
...
-	font-family: "Roboto Slab", serif;
+	font-family: "Nunito Sans", serif;
 	font-size: 1.2em;
 	font-weight: lighter;
 	letter-spacing: 3px;
 	color: #cccccc;
 	-webkit-font-smoothing: antialiased; }
 
+.crown-logo {
+	width: 100px;
```
Comment (Josh-00FF00) [inline _sass/uno.scss]: Can we make this percentage based?

File: _sass/uno.scss [hunk 12]
```diff
@@ -1452,16 +1459,16 @@ i {
 	color: #c7c7c7; }
 
 .post-list__meta--date {
-	margin-right: .5em;
+	text-transform: lowercase;
```
Comment (Josh-00FF00) [inline _sass/uno.scss]: Same about dates as above

File: images/favicons/manifest.json [hunk 13]
```diff
@@ -1,41 +1,41 @@
...
+  },
+  {
+   "src": "\/android-icon-96x96.png",
+   "sizes": "96x96",
+   "type": "image\/png",
+   "density": "2.0"
+  },
+  {
+   "src": "\/android-icon-144x144.png",
+   "sizes": "144x144",
```
Comment (Josh-00FF00) [inline images/favicons/manifest.json:30]: Good shout on the formatting, it is a bit wild at 4 spaces

File: _sass/uno.scss [hunk 14]
```diff
@@ -1108,10 +1107,12 @@ pre {
 
 .footer {
 	display: block;
-	padding: 2em 0 0 0;
-	border-top: 2px solid #dddddd;
+	padding: 17px 0 6px 0;
```
Comment (Josh-00FF00) [inline _sass/uno.scss]: Can these be em values?

File: _sass/uno.scss [hunk 15]
```diff
@@ -1108,10 +1107,12 @@ pre {
...
+	background: #f9f9f9;
 	font-size: .7em;
-	color: #b3b3b3; }
+	margin-left: 530px;
```
Comment (Josh-00FF00) [inline _sass/uno.scss]: Does this need to be fixed? Should this vary for screen size?

**Review by Josh-00FF00 — CHANGES_REQUESTED** [review]
Looking good, like the new images and colour changes. Few bits to look at first though

File: _config.yml (hunk 1 above)
Comment (jmhrpr) [reply _config.yml]: I'm not sure where I took this slogan from, but it was something already in use - it helps people who might not know what a CTF is?

File: _includes/header.html (hunk 2 above)
Comment (jmhrpr) [reply _includes/header.html:80]: Does anyone use it? Clicking it takes me to an unstyled mess. Can't we just keep the functionality without having the button in such an obvious place?

File: images/favicons/browserconfig.xml (hunk 3 above)
Comment (jmhrpr) [reply images/favicons/browserconfig.xml]: This is generated by image->favicon converter

File: _config.yml (hunk 1 above)
Comment (Josh-00FF00) [reply _config.yml]: https://github.com/cr0wnctf/cr0wnctf.github.io/commit/5293e77c236a7151ed15a81aa7e20d736994f180#diff-aeb42283af8ef8e9da40ededd3ae2ab2R3

Probably remembered it from here. I think it's punchier being shorter I'll throw it to the committee and we can debate it for a week

File: _includes/header.html (hunk 2 above)
Comment (Josh-00FF00) [reply _includes/header.html:80]: Dunno about usage, we have no stats (something I'll fix soon). But yeah moving it to a smaller link down below is a good compromise

File: _config.yml (hunk 1 above)
Comment (jmhrpr) [reply _config.yml]: The terms all being so short puts me off, the variation in word length is nicer imo.

File: images/favicons/browserconfig.xml (hunk 3 above)
Comment (Josh-00FF00) [reply images/favicons/browserconfig.xml]: Can chuck it through `xmllint`

File: _includes/header.html (hunk 5 above)
Comment (jmhrpr) [reply _includes/header.html:12]: Should be there now

File: _sass/uno.scss (hunk 9 above)
Comment (jmhrpr) [reply _sass/uno.scss]: Removed

File: _sass/uno.scss (hunk 8 above)
Comment (jmhrpr) [reply _sass/uno.scss]: All the tags are lowercase, stylistically the branding is lowercase so I would argue it looks out of place to be capitalised, but we're talking about a single, low-opacity character - does it matter?

File: _sass/uno.scss (hunk 10 above)
Comment (jmhrpr) [reply _sass/uno.scss:1193]: Looks goofy

File: _includes/header.html (hunk 2 above)
Comment (jmhrpr) [reply _includes/header.html:80]: Will add RSS again somewhere

File: _sass/uno.scss (hunk 8 above)
Comment (Josh-00FF00) [reply _sass/uno.scss]: Is a proper noun, thought it looked amatuerish like we failed to write it properly. But yeah not a hill I'll die on if you strongly advocate it

"12 jun 2019" vs "12 Jun 2019"

Comment (jmhrpr) [reply _sass/uno.scss]: With everything else lowercase I think the negatives of people thinking we can't capitalise are removed, so ok to take the positive of stylistic consistency.

File: _sass/uno.scss (hunk 11 above)
Comment (jmhrpr) [reply _sass/uno.scss]: I'm not sure that is really correct, the title text below isn't based on window size and the logo size is chosen relative to that, I'm not sure it should be done on percentage?

Comment (jmhrpr) [reply _sass/uno.scss]: I'm not sure that is really correct, the title text below isn't based on window size and the logo size is chosen relative to that, I'm not sure it should be done on percentage?

File: _sass/uno.scss (hunk 15 above)
Comment (jmhrpr) [reply _sass/uno.scss]: This footer was giving me a lot of grief. I've reached a compromise.

File: _sass/uno.scss (hunk 14 above)
Comment (jmhrpr) [reply _sass/uno.scss]: I think the footer issues are resolved now.

File: _config.yml (hunk 1 above)
Comment (Josh-00FF00) [reply _config.yml]: No response from the peeps, I'm happy to keep this but I think it should be Title Capitalisation

File: index.html [hunk 16]
```diff
@@ -11,9 +11,14 @@
         <h2 class="post-list__post-title post-title"><a href="{{ site.baseurl }}{{ post.url }}" title="{{ post.title }}">{{ post.title }}</a></h2>
         <p class="excerpt">{{ post.excerpt | strip_html }}&hellip;</p>
         <div class="post-list__meta">
-            <time datetime="{{ post.date | date: "%Y-%m-%d %H:%M" }}" class="post-list__meta--date date">{{ post.date | date: "%-d %b %Y" }}</time>
+            <time datetime="{{ post.date | date: "%Y-%m-%d %H:%M" }}" class="post-list__meta--date date">
+            {% if post.authors.size > 0 %}
+            by <span style="font-weight: bold; color:#868686">
+            {{ post.authors | array_to_sentence_string }}</span> on
```
Comment (Josh-00FF00) [inline index.html:17]: Like this very much good work

File: _includes/header.html (hunk 2 above)
Comment (Josh-00FF00) [reply _includes/header.html:80]: Not resolved just yet

File: _config.yml (hunk 1 above)
Comment (jmhrpr) [reply _config.yml]: Capitalised description line

File: _includes/header.html (hunk 2 above)
Comment (jmhrpr) [reply _includes/header.html:80]: It's now in the footer, where it belongs!

File: _sass/uno.scss (hunk 14 above)
Comment (Josh-00FF00) [reply _sass/uno.scss]: image

Not quite there yet. This happens when the view switches to responsive

**Review by Josh-00FF00 — APPROVED** [review]

**Pull request merged**